blob/service.go

```diff
@@ -86,6 +89,47 @@ func DefaultSubmitOptions() *SubmitOptions {
 	}
 }
 
+type BlobsubResponse struct {
```

Comment:
we should move away from these kind of names, it'll be kinder to everyone that uses these types by giving them descriptive names, if boring. This looks like a `SubscriptionResponse`, as it's in the blob module, we know it's the `blob.SubscriptionResponse`, that'd be nice to read and think about.

Reply:
definitely, nice catch